third_party/nvfuser/test/test_gpu3.cpp

```diff
@@ -1436,6 +1436,18 @@ TEST_F(NVFuserTest, FusionCodegenAllocatedScalars_CUDA) {
                << indent << tk0_name << "[(0 + 1)]\n"
                << indent << indent << " = " << tk0_name << "[((0 + 1) + 1)];\n";
 
+  // This test fails with eager constant addition:
+  //   C++ exception with description "Invalid code generation. Expected:
+  //       T0[(0 + 1)]
+  //            = T0[((0 + 1) + 1)];
+  //   Actual:
+  //   __global__ void CUDAGeneratedKernel(Tensor<float, 0> T0, Tensor<float, 0>
+  //   T1) {
+  //       T1[0]
+  //              = T0[0];
+  //         T0[1]
+  //                = T0[2];
+  //   }
```

Comment:
This test fails because with eager constant ops, `Allocated` still results in a literal being inserted. Converting the input `zeroVal()` to a non-constant `Scalar` changes the character of the test, so I am not sure how to modify this test to accomodate eager constant ops.